mobify/platform-scaffold
# Fix sfcc cart item update

This PR moves the connector registration into a separate file, so we can make sure the connector extension is only being registered for the merlins connector. This problem is pretty specific to the scaffold, so we may want to look into how we could revert this in the generator. 

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-124
 **Linked PRs**: n/a

## Changes
- Moved connector registration for the merlins and SFCC connectors into separate files that are imported into main.jsx

## How to test-drive this PR
- switch the import statements in main.jsx to import the sfcc connector config function
- Run `npm run dev`
- Preview the [SFCC site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to your cart
- update the item's quantity from the cart page
- the order totals should updated correctly
